cmd/govim/buffer_events.go

```diff
@@ -28,8 +28,19 @@ func (v *vimstate) bufReadPost(args ...json.RawMessage) error {
 
 	if cb, ok := v.buffers[nb.Num]; ok {
 		// reload of buffer, e.v. e!
-		cb.SetContents(nb.Contents())
 		cb.Loaded = nb.Loaded
+
+		// If the contents is the same we probably just re-loaded an already open buffer.
```

Comment:
"If the contents _are_ the same..."

I don't think it's strictly correct to say "open buffer" either. Would it perhaps be more accurate to say "... probably just re-loaded a currently unloaded buffer"?

Reply:
Good point, updated!